### PR TITLE
Raspberry PI Camera Demo - Added Note to Step 9

### PR DESCRIPTION
**Pull request by WaywardHayward**
I added a note to step 9 as missing this caused an afternoon of searching for resolutions

### CODE REVIEW

File: docs/demos/usb-camera-demo-rpi4.md [hunk 1]
```diff
@@ -40,6 +40,13 @@ The following will be covered in this demo:
     ```sh
     kubectl taint nodes --all node-role.kubernetes.io/master-
     ```
+     > **Note** If this command does not work you may need to setup kubenetes config and environment variables using the commands below
```
Comment (kate-goldenring) [inline docs/demos/usb-camera-demo-rpi4.md:43]: Can we move these above the `kubectl taint` as these first three lines are always required and are requested actions at the end of kubeadm init?

**Review by kate-goldenring — COMMENTED** [review]
Thanks for adding this!

File: docs/demos/usb-camera-demo-rpi4.md (hunk 1 above)
Comment (WaywardHayward) [reply docs/demos/usb-camera-demo-rpi4.md]: Moved to the init step.

**Comment by kate-goldenring** [comment]
@WaywardHayward this is great. Just need to sign the commits and its good to go. To sign:
```
git rebase HEAD~2 --signoff
git push --force-with-lease origin main
```

**Review by kate-goldenring — APPROVED** [review]

**Pull request merged**